cookbook/bundles/tests.md

```diff
@@ -0,0 +1,255 @@
+.. index::
+   single: Bundle; Testing third party bundles
+
+Writing tests for third party bundles
```

Comment:
"Writing Tests for Third Party Bundles" (we follow the "capitialize all words, except from closed-class words" title case standard)